src/Pages/SingleArticle.jsx

```diff
@@ -0,0 +1,56 @@
+import { React, useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { getArticle } from '../utils/api';
+import dayjs from 'dayjs';
+
+const SingleArticle = () => {
+    const { id } = useParams();
+    const [article, setArticle] = useState({});
+    const [isLoading, setIsLoading] = useState(true);
+    const [error, setError] = useState(false);
+
+    useEffect(() => {
+        getArticle(id).then(({ data }) => {
+            setArticle(data);
+        })
+        .catch(err => {
+            setError(true);
+            setIsLoading
+        })
+    }, [id]);
+
+    if (error) return <h1>Something went wrong!</h1>;
+
+    return (
+        <div>
+            <h2>{article.title}</h2>
+            <p>
+                <span style={{ fontWeight: 'bold' }}>Topic: </span>
```

Comment:
As all of these spans have the same styling, we can provide them the same classname and have the styling in the css file